src/pages/igv/types/ListBucket.inerface.ts

```diff
@@ -0,0 +1,49 @@
+interface Key {
+  _text: string
+}
+
+interface LastModified {
+  _text: Date
+}
+
+interface ETag {
+  _text: string
+}
+
+interface Size {
+  _text: string
+}
+
+interface ID {
+  _text: string
+}
+
+interface DisplayName {
+  _text: string
+}
```

Comment:
define a common interface 
interface IText: {
  _text: string
} and replace all duplicates with it.